imports/api/blocks/server/methods.js

```diff
@@ -628,38 +597,7 @@ Meteor.methods({
                         getValidatorUptime(validatorSet)
                     }
 
-                    // fetching keybase every base on keybaseFetchingInterval settings
-                    // default to every 5 hours 
 
```

Comment:
@MonikaCat if you remove this piece of code, will the profile pictures be fetched when the validators are first being indexed?

Reply:
You are right, my bad! I have updated and tested the code, it will fetch avatars when the validators are indexed for the first time and then it will update it on every keybaseFetchingInterval value (in our case every 5 hours).